Description: Debian doesn't have SSLv3 symbols in Python anymore.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2015-03-25

--- openstack-trove-2015.1~b3.orig/trove/openstack/common/sslutils.py
+++ openstack-trove-2015.1~b3/trove/openstack/common/sslutils.py
@@ -82,8 +82,7 @@ def wrap(sock):
 
 _SSL_PROTOCOLS = {
     "tlsv1": ssl.PROTOCOL_TLSv1,
-    "sslv23": ssl.PROTOCOL_SSLv23,
-    "sslv3": ssl.PROTOCOL_SSLv3
+    "sslv23": ssl.PROTOCOL_SSLv23
 }
 
 try:
