Description: Fix import error
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2016-01-23

Index: openstack-trove/trove/tests/unittests/trove_testtools.py
===================================================================
--- openstack-trove.orig/trove/tests/unittests/trove_testtools.py
+++ openstack-trove/trove/tests/unittests/trove_testtools.py
@@ -143,7 +143,7 @@ class TestCase(testtools.TestCase):
                                 full_name, member, container, depth + 1)
             except ImportError:
                 pass  # Module cannot be imported - ignore it.
-            except RuntimeError:
+            except:
                 # Something else went wrong when probing the class member.
                 # See: https://bugs.launchpad.net/trove/+bug/1524918
                 pass
