Description: Fixes keystone credentials in the default config file.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2013-10-23

--- openstack-trove-2013.2.orig/etc/trove/api-paste.ini
+++ openstack-trove-2013.2/etc/trove/api-paste.ini
@@ -18,6 +18,9 @@
 auth_host = 127.0.0.1
 auth_port = 35357
 auth_protocol = http
+admin_tenant_name = %SERVICE_TENANT_NAME%
+admin_user = %SERVICE_USER%
+admin_password = %SERVICE_PASSWORD%
 # signing_dir is configurable, but the default behavior of the authtoken
 # middleware should be sufficient.  It will create a temporary directory
 # in the home directory for the user the trove process is running as.
