Description: Make trove-taskmanager.conf parseable
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2016-03-31

--- openstack-trove-5.0.0~rc1.orig/etc/trove/trove-taskmanager.conf.sample
+++ openstack-trove-5.0.0~rc1/etc/trove/trove-taskmanager.conf.sample
@@ -167,7 +167,6 @@ pydev_debug = disabled
 # registry server. Any valid SQLAlchemy connection string is fine.
 # See: http://www.sqlalchemy.org/docs/05/reference/sqlalchemy/connections.html#sqlalchemy.create_engine
 connection = mysql://root:e1a2c042c828d3566d0a@localhost/trove
-# connection = mysql://root:root@localhost/trove
 
 # Period in seconds after which SQLAlchemy should reestablish its connection
 # to the database.
