Description: Better default in the config file
Author: Thomas Goirand <zigo@debian.org>
Forwarded: not-needed
Last-Update: 2013-10-03

Index: openstack-trove/etc/trove/trove.conf.sample
===================================================================
--- openstack-trove.orig/etc/trove/trove.conf.sample	2014-07-01 23:28:54.000000000 +0800
+++ openstack-trove/etc/trove/trove.conf.sample	2014-07-02 15:11:12.000000000 +0800
@@ -45,9 +45,7 @@
 # SQLAlchemy connection string for the reference implementation
 # registry server. Any valid SQLAlchemy connection string is fine.
 # See: http://www.sqlalchemy.org/docs/05/reference/sqlalchemy/connections.html#sqlalchemy.create_engine
-# sql_connection = sqlite:///trove_test.sqlite
-sql_connection = mysql://root:e1a2c042c828d3566d0a@localhost/trove
-#sql_connection = postgresql://trove:trove@localhost/trove
+sql_connection = sqlite:///var/lib/trove/trovedb
 
 # Period in seconds after which SQLAlchemy should reestablish its connection
 # to the database.
